h/oauth/jwt_grant.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, oauth_validator, user_svc, domain):
 
 class JWTAuthorizationGrant(GrantTypeBase):
     def __init__(self, request_validator, user_svc, domain):
-        self.request_validator = request_validator
+        super().__init__(request_validator)
```

Comment:
https://github.com/oauthlib/oauthlib/blob/master/oauthlib/oauth2/rfc6749/grant_types/base.py

Reply:
Looks like this one could have been important.